TSC-Charter.md

```diff
@@ -36,18 +36,19 @@ approved by the Board.
 
 ## Section 4. Establishment of the TSC.  
 
-TSC memberships are not time-limited. There is no fixed size of the TSC.
-However, the expected target is between 6 and 12, to ensure adequate
-coverage of important areas of expertise, balanced with the ability to
-make decisions efficiently.
+TSC memberships are not time-limited. There is no maximum size of the TSC.
+The size is expected to vary in order to ensure adequate coverage of important
+areas of expertise, balanced with the ability to make decisions efficiently.
+The TSC must have at least three members.
```

Comment:
A good question indeed. I would guess board intervention of some sort, perhaps dissolution of the remaining TSC?

Thinking out loud here, I don't think this is likely to happen, but if it were, I suspect that would mean something has gone really wrong and the project would be without leadership. I'm not saying that the TSC is itself project leadership, but that many people leaving the TSC would probably be because of some sort of exodus on a larger scale. In other words, I think it's really unlikely for this event to occur in a vacuum.